xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/locationPicker_macros.vm

```diff
@@ -242,7 +244,11 @@
             'showTerminalDocuments': $showTerminalDocuments,
             'showTranslations': false,
             'showWikis': $showWikis,
-            'exclusions': "$!options.exclusions"
+            'exclusions': "$!options.exclusions",
+            'root': "$!options.root",
+            'showSpaces': $macro.showSpaces,
```

Comment:
I'm a bit surprised by this. In XWiki Standard we're using by default the **nested pages** hierarchy, meaning that spaces are not shown in the UI normally. The user interacts only with pages. IIUC, you're using ``showSpaces=true`` in your extension, but how does this reconcile with the rest of the XWiki UI (where your extension is installed) that doesn't show spaces but nested pages?

Reply:
Thank you for your review.

The reason why I set `showSpaces` to true is because I would like to have selector like this:
![image](https://github.com/user-attachments/assets/59b897f6-5799-4870-acbd-9473a40eb1ff)

And I put `showSpaces` to false I end with something like this:
![Screenshot_20250428_171017](https://github.com/user-attachments/assets/310033e2-7e9d-432a-97de-0a56f14e209d)

So I don't really exactly know what is happening around this but in this case it seem that this parameter solve what I need.

Reply:
So, after investigating I've found the correct way to configure the picker without `showSpaces=true`. So I've removed this depreciated parameter.